### ISSUE TITLE
[🐛 bug report] Error when compiled with gpu variants - error while loading shared libraries: libcuda.so.1 

### ISSUE THREAD

**Issue by Spandan-Madan** [mitsuba-renderer/mitsuba2#298]
## Summary

I installed the gpu_autodiff_rgb variant and am trying to launch mitsuba but I get this error both when I try to run mitsuba from command line and while importing mitsuba in python:

mitsuba: error while loading shared libraries: libcuda.so.1: cannot open shared object file: No such file or directory

Any suggestions what might be wrong?

Thanks in advance!

## System configuration

 For reproducing - I'm running this docker container I created: https://hub.docker.com/repository/docker/spandanmadan/mitsuba2_gpu
- Compiled variants:
    * `scalar_rgb`
    * `packet_rgb`
    * `gpu_autodiff_rgb`

**Comment by Speierers** [comment]
Hi @Spandan-Madan ,

What's the cuda version installed on your system?

**Comment by Spandan-Madan** [comment]
By default it's CUDA 10.1, but I also tried with CUDA 11.1, got same errors.

**Comment by Spandan-Madan** [comment]
I realized that I was running docker container with the wrong flags and so it didn't have access to cuda files. Upon correcting, I'm getting a new error:-

`cuda_check(): runtime API error = 0999 "cudaErrorUnknown" in ../ext/enoki/src/cuda/jit.cu:279.`

**Comment by Speierers** [comment]
And which version of CUDA are you running on this system?

**Comment by Spandan-Madan** [comment]
Running CUDA 10.1! I tried to generate the PTX kernels myself as well. For this, I have downloaded and unpacked Optix 7.2 from NVIDIA's website into `/opt/NVIDIA-OptiX-SDK-7.2.0-linux64-x86_64', and set the path of that folder in the `resources/ptx/Makefile'. Then, when I try to run `make` from the PTX folder, I got this:

```
nvcc ../../src/librender/optix/optix_rt.cu \
	 -I ../../include -I ../../src/shapes/optix -I /opt/NVIDIA-OptiX-SDK-7.2.0-linux64-x86_64/include/ \
	 -O3 -gencode arch=compute_61,code=compute_61 --ptx
```

But I still get the same error.
